src/card.css

```diff
@@ -0,0 +1,115 @@
+@import "/product_card-styles/stars.css";
+
```

Comment:
no, move styles that refer to the card in the file card.css, and those that are general in keep in main.css